api/mapping/models.py

```diff
@@ -226,8 +229,9 @@ class DocumentFile(BaseModel):
             on_delete=models.CASCADE,
             blank=True,
             null=True)
+    status=models.CharField(max_length=20,choices=STATUS_CHOICES)
```

Comment:
Please, define one of the options as the default option. It will also help the migrations.